### ISSUE TITLE
Newly trained Lora generates random noise.

### ISSUE THREAD

**Issue by jkindwall** [bmaltais/kohya_ss#1297]
Hi,
I'm new to Kohya_ss and lora training in general, though I have trained a few textual inversion embeddings.  I just tried following this tutorial: https://www.youtube.com/watch?v=70H03cv57-o, and after only a slight hiccup with the mixed_precision/save_precision settings I was able to get a training session to run to completion.  However,

My first attempt was with the Basic settings config file linked from that youtube video.  This produced a lora that when used gave me the following error when I tried to generate images:

NansException: A tensor with all NaNs was produced in Unet

So, I tried again using the Low VRAM settings file also linked in the video (I'm using an RTX 2080 with 8GB), and this time I get no errors when generating images, but everything I generate just ends up as random noise:

![grid-2107](https://github.com/bmaltais/kohya_ss/assets/2964211/b53f4680-bbf8-4927-8fa7-9d1c5f7ec266)

Could this be related to this issue: https://github.com/bmaltais/kohya_ss/issues/1291? 

Or do I have bigger problems?

**Comment by IndolentKheper** [comment]
The Aitrepeneur tutorial is outdated and the settings just aren't good. It's better to set low repeats and raise the number of epochs. Also, settings files made in older versions of Kohya result in broken LoRAs for some reason, so try inputting the settings manually. 8GB VRAM should be enough to train a 1.5 LoRA. Batch size 1, fp16, 10 epochs, default learning rates, constant or cosine, Adamw8bit, network dim/alpha 32/16, input those settings manually on a new loadout, don't load any old .json config files. Change the folder name on your images to 10_ instead of Aitrepeneur's loony 100_ which generally isn't a good way to go, you'll likely overbake the LoRA. Splitting it up over 10-15 epochs will give you a batch of LoRAs to test and find out when it started overbaking. 

I think mainly you got that result from using an old config, that happened to me when I updated Kohya and tried to use my 21.5 configs with it. Otherwise, if you notice loss=nan during training then you're definitely gonna get broken LoRAs like these, so you'll have to figure out what's causing that and fix that too.

**Comment by jkindwall** [comment]
Hi @IndolentKheper,

Thanks for the advice.  Actually, I tried downgrading to v21.7.10 per @JeroenHeyndrickx instructions here: https://github.com/bmaltais/kohya_ss/issues/1291#issuecomment-1657098477.

I also downgraded from PyTorch 2 to PyTorch 1, and now I'm able to generate working LoRAs, although I think you're right that they're overbaked.  I need to set a fairly low Cfg Scale to get a decent likeness in the face, and they're really inflexible.  Every image seems to be a similar pose, from a similar angle with similar clothes, and a similar background.  Whereas when I use the same prompts with just the base model, I get a much more dynamic range of images.

I'll try your inputting your suggested settings manually, and see how it goes.
